app/views/admin/services/_service.html.erb

```diff
@@ -4,6 +4,13 @@
     <% if service.active_data_set %><span class="text-muted"> <abbr title="Version <%= service.active_data_set_version %>">#<%= service.active_data_set_version %></abbr></span>
     <% end %>
   </td>
+  <td>
+    <% if lookup.govuk_page?(service.slug) %>
```

Comment:
Minor: Indentation missing from this block

Reply:
Fixed!